Dockerfile.build

```diff
@@ -0,0 +1,7 @@
+FROM gradle:7.2.0-jdk11-alpine AS build
+
+COPY --chown=gradle:gradle . /src
+
+WORKDIR /src
+
+ENTRYPOINT ["gradle", "build"]
```

Comment:
What will happen if your project would migrate to the newer version of the Gradle?